packages/ui-react/src/components/Account/Account.tsx

```diff
@@ -261,6 +267,10 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
           formSection({
             label: t('account.email'),
             onSubmit: async (values) => {
+              if (!(values.email && values.confirmationPassword)) {
+                // This is a workaroud to make the form accesible when a empty form has been submitted.
+                throw new Error('Invalid confirmationPassword');
```

Comment:
This is an error that the user will see right? If so, we need to update the text to a more descriptive text for the user. We can re-use the other translation 'Password incorrect!" for this

Reply:
Should it be "Missing password/email" instead? It now seems like the user has entered a password, but it's incorrect. In reality, no password or email was given at all.

Reply:
Ah yes, that is indeed a better approach.
Which is also in line with the general consensus that we actually shouldn't have error messages indicating that only the password (or email) is wrong, to prevent someone from trying to brute force an account by 'guessing' the password since they know only the password is incorrect (thus the email correct). But that's a discussion for another time, and more applicable to the sign up modal.

Reply:
I wanted to reuse an existing error message from the `switch` present Account.tsx. This one seems to be the most closest one. 'Password incorrect!" doesn't exists. So I need to do add my own. This will not make it a workaround but a really ugly fix in my eyes 😅 

`throw new Error('Wrong email/password combination');`

and this:
```
case 'Wrong email/password combination': {
            formErrors.form = t('account.login.wrong_combination');
            break;
          }
          ```
          
Is this worth the effort though?

Reply:
I think it's worth an extra switch case instead of letting users see the 'Invalid confirmationPassword' error. Does Chris agree?

Reply:
Even though I think it's an edge case users will barely see, I applied my suggested change. I know it can be a lot prettier, but i tried to keep it simple. The whole logics of form error handling should be rewritten preferably and more inline with other form validation